src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -860,5 +885,25 @@ private void InitializeAnimation()
                 newAnimation = null;
             }
         }
+
+        // Show and Register Content of Page to Accessibility bridge
+        private void ShowContentOfPage(Page page)
```

Comment:
I think this design seems a bit of an abuse of authority.
If we want to request to show or to hide Content of each Page, I think it is better to make ShowContent() and HideContent() methods in Page and call them here.
Page.ShowContent() and HideContent() may become virtual.
And real implementation will be written in each inherit class.

Reply:
agreed.

Reply:
Regarding this, we need a separate patch because Page needs Content then, as you know.
Can we go to another PR about this one?

Reply:
I agree with @Seoyeon2Kim .
If @bshsqa  and @everLEEst  agrees, we can make a separate PR.